backend/lib/utils.js

```diff
@@ -429,4 +429,12 @@ module.exports = {
       ;`;
     await module.exports.dbParamQuery(client, query, data, loggerOptions);
   },
+  getEnabledCrawlers: (crawlersConfig) => (
+    crawlersConfig.filter((crawlerConfig) => (
+      crawlerConfig.enabled === true
+    ))
+  ),
+  getCrawlerNames: (crawlersConfig) => (
```

Comment:
Please, refactor to have `getEnabledCrawlersNames`

Reply:
updated